docs/tutorial.md

```diff
@@ -0,0 +1,183 @@
+# Oria Operator Tutorial
+
+### An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.8.0 or newer
```

Comment:
```suggestion
- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.21.0 or newer
```

Probably best to just pick a newer one to avoid hitting any issues with older versions.

Reply:
agree